### PR TITLE
[tw] Localize Monolithic Apps for Traditional Chinese

### LINKED ISSUE #2539: [tw] Localize `Monolithic Apps` for Traditional Chinese
1. Localize `Monolithic Apps` for Traditional Chinese
2. https://glossary.cncf.io/monolithic-apps/

### PR DESCRIPTION
**Pull request by johnlinp**
### Describe your changes

Origin content from https://glossary.cncf.io/monolithic-apps/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2539

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/monolithic-apps.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: 單體式應用程式
+status: Completed
+category: 概念
+tags: ["架構", "基本原理", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+單體式應用程式是一個可部署的程式，且包含所有功能在其中。
```
Comment (hwchiu) [inline content/zh-tw/monolithic-apps.md:10]: 我覺得這個翻譯真的難XD
AWS: 巨型
https://aws.amazon.com/tw/compare/the-difference-between-monolithic-and-microservices-architecture/

Azure: 整合型應用程式
https://learn.microsoft.com/zh-tw/azure/architecture/microservices/migrate-monolith

GCP: 單體
https://cloud.google.com/resources/rearchitecting-to-cloud-native?hl=zh-tw

單體式 單套式 等翻譯網路上都找得到類似的文章，似乎沒有一個完全統一的標準

Comment (hwchiu) [reply content/zh-tw/monolithic-apps.md:10]: 我 slack 上開了一個討論串來討論看看採用哪種名稱比較好

https://cloud-native.slack.com/archives/C055KERTNCX/p1697858155551159

Comment (johnlinp) [reply content/zh-tw/monolithic-apps.md:10]: 根據 slack 上的討論，我們決定維持現在 "單體式應用程式" 這個翻譯，謝謝。

Comment (ydFu) [reply content/zh-tw/monolithic-apps.md:10]: 單體式+1，基於上述討論基礎進行支持。
補充另外的支持論點是「單體式」的傳唱性在繁體中文也蠻高的。

**Review by hwchiu — APPROVED** [review]
LGTM

**Review by pichuang — APPROVED** [review]
LGTM

**Pull request merged**